web.md

```diff
@@ -0,0 +1,37 @@
+# Web
+
+**Prerequisite knowledge**: 
+
+## Intro
+Web - short for World Wide Web.
+
+No prerequisite knowledge required. If you see this document, you are probably aware of what computers are and that they can communicate with each other via global network of computer networks, called the Internet. If so, you're all set to begin your journey with the web.
+
+## Classification / acceptance questions
+### Apprentice:
+- I can tell what World Wide Web is.
+- I can tell the difference betweend web and the internet.
+- I know what a web browser is.
+- I know what a web server is.
+
+### Intermediate
+- I can explain how does the Web work.
+- I know what DNS is.
+- I can describe step by step what happens when I put `www.youtube.com` into the address bar and hit enter before I can see the results.
+- I can make the above happen for my own custom domain name, which includes:
+  - regestering a domain name
+  - adding a DNS record, that will point the domain to a
+  - web server I've set up, which will serve some
+  - HTML files I uploaded.
+
+### Advanced
+
+
+## Sources
```

Comment:
What do you think about adding an article from wikipedia: https://en.wikipedia.org/wiki/World_Wide_Web#Dynamic_updates_of_web_pages for dinosuars like me that prefer text over video? :)

Reply:
https://dyn.com/blog/dns-why-its-important-how-it-works/